package.json

```diff
@@ -19,7 +19,7 @@
         "axios": "^0.27.2",
         "bootstrap": "^5.1.2",
         "clipboard": "^2.0.6",
-        "cosmjs-types": "^0.5.0",
+        "cosmjs-types": "0.4.0",
```

Comment:
@jay-dee7 Is there a reason for the downgrade?

Reply:
@Eengineer1 that other cosmjs libs that we use (like `proto-signing` or `stargaze`) use cosmjs-types@0.4.0 which so using v0.5.0 causes type conflicts. They're switching to v0.5 of cosmjs-types in v0.29 of @cosmjs/* packages, so we can upgrade this then